test/classes.test.js

```diff
@@ -0,0 +1,35 @@
+import Project from '../src/addproject';
+import Task from '../src/addtask';
+import Library from '../src/library';
+
+const library = new Library();
+const project = new Project('test');
+const task1 = new Task('test', 'low', '12/03/2000', 'This is a test');
+it('check if a Project can be created', () => {
+  expect(project).toBeDefined();
+});
+
+it('check if the Project title is the same than the input', () => {
+  expect(project.title).toBe('test');
+});
+
+it('check if a Task can be created', () => {
+  expect(task1).toBeDefined();
+});
+
+it('check if Task take the correct input', () => {
+  expect(task1.title).toBe('test');
+  expect(task1.priority).toBe('low');
+  expect(task1.date).toBe('12/03/2000');
+  expect(task1.description).toBe('This is a test');
+});
+
+it('check if a project have a task', () => {
+  project.tasks.push(task1);
+  expect(project.tasks[0]).toBeDefined();
+});
+
+it('check if a project can be saved', () => {
+  library.myprojects.push(project);
+  expect(library.myprojects[0]).toBeDefined();
+});
```

Comment:
According to the pathwright, we need to create a separate test file for each class or module.
And we need to write a test for all our functions.

![image](https://user-images.githubusercontent.com/31897434/98059092-8dc56500-1e46-11eb-9bfd-5603c93f7771.png)
https://microverse.pathwright.com/library/javascript-module-v10-copy/166786/path/step/97635179/